tests/components/mqtt/test_light.py

```diff
@@ -1185,7 +1185,7 @@ async def test_on_command_rgb(hass, mqtt_mock):
 
     mqtt_mock.async_publish.assert_has_calls(
         [
-            call("test_light/rgb", "1,0,0", 0, False),
+            call("test_light/rgb", "1,1,0", 0, False),
```

Comment:
Why does this change?

Reply:
The colors channels are now rounded instead of being truncated, which should be more correct.
There's a lot of truncating going on in util/color.py, so maybe we should not change the behavior in this PR but instead consider that for a separate PR?

Reply:
If you're ok with it I'm ok with it. It's just a red flag for me when tests need to change during refactors. That's why I asked.